.rubocop.yml

```diff
@@ -30,12 +30,18 @@ Metrics/MethodLength:
   Max: 30
 Metrics/AbcSize:
   Max: 30
+
+Naming/AccessorMethodName:
+  Enabled: false
```

Comment:
I assume this is for a function we're overriding and can't rename? Could you disable it only for the file containing the function?

Reply:
Didn't know we could do that. Done.